model_analyzer/result/sorted_results.py

```diff
@@ -35,53 +40,55 @@ def empty(self):
         """
         Returns
         -------
-        True if this heap has no results
+        True if this list has no results
         False otherwise
         """
 
         return not bool(self._sorted_results)
 
-    def results(self):
+    def results(self) -> List[RunConfigResult]:
         """
         Returns
         -------
-        All the results in this result heap
+        All the results in this result list
```

Comment:
refer to the class/object, not to "this list"

Reply:
Fixed